rootex/framework/components/transform_component.cpp

```diff
@@ -7,10 +7,19 @@
 
 Component* TransformComponent::Create(const JSON::json& componentData)
 {
+	BoundingBox boundingBox({ 0.0f, 0.0f, 0.0f }, { 0.5f, 0.5f, 0.5f });
+	if (componentData.contains("boundingBox"))
```

Comment:
i suppose this was for debugging?

Reply:
No, it's so that we don't have to edit component data manually